tests/tests.js

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable no-undef*/
+describe('String Class Extra Methods', () => {
+  describe('hasVowels', () => {
+    const actuals = {
+      hasVowels1: 'Testing',
+      hasVowels2: 'tstIng',
+      hasVowels3: 'tstng'
+    };
+    it('Should return True if the string contains vowels', () => {
+      expect(actuals.hasVowels1.hasVowels()).toBeTruthy();
+      expect(actuals.hasVowels2.hasVowels()).toBeTruthy();
+    });
+    it('Should return False if the string does not contain vowels', () => {
+      expect(actuals.hasVowels3.hasVowels()).toBeFalsy();
+    });
+  });
+  describe('toUpper', () => {
+    const actuals = {
+      toUpper1: 'Testing',
+      toUpper2: 'tsTing',
+      toUpper3: 'JINGLE',
+      toUpper4: 'jingl55E'
+    };
+    it('Should return uppercase string for all passed in data', () => {
+      expect(actuals.toUpper1.toUpper()).toEqual('TESTING');
+      expect(actuals.toUpper2.toUpper()).toEqual('TSTING');
+      expect(actuals.toUpper3.toUpper()).toEqual('JINGLE');
+      expect(actuals.toUpper4.toUpper()).toEqual('JINGL55E');
+    });
+  });
+  describe('toLower', () => {
+    const actuals = {
+      toLower1: 'Testing',
+      toLower2: 'tsTing',
+      toLower3: 'JINGLE',
+      toLower4: 'jingl55E'
+    };
+    it('Should return a lowercase string for all passed in strings', () => {
+      expect(actuals.toLower1.toLower()).toEqual('testing');
+      expect(actuals.toLower2.toLower()).toEqual('tsting');
+      expect(actuals.toLower3.toLower()).toEqual('jingle');
+      expect(actuals.toLower4.toLower()).toEqual('jingl55e');
+    });
+  });
+  describe('ucFirst', () => {
+    const actuals = {
+      ucFirst1: 'Testing',
+      ucFirst2: 'tsTing',
+      ucFirst3: 'jingle'
+    };
+    it('Should return string with first character in uppercase', () => {
+      expect(actuals.ucFirst1.ucFirst()).toEqual('Testing');
+      expect(actuals.ucFirst2.ucFirst()).toEqual('TsTing');
+      expect(actuals.ucFirst3.ucFirst()).toEqual('Jingle');
+    });
+  });
+  describe('isQuestion', () => {
+    const actuals = {
+      isQuestion1: 'Testing?',
+      isQuestion2: 'tsTing isn\'t it?',
+      isQuestion3: 'JINGLE',
+      isQuestion4: 'JING?le?'
+    };
+    it('Should return true if a string is a question', () => {
+      expect(actuals.isQuestion1.isQuestion()).toBeTruthy();
+      expect(actuals.isQuestion2.isQuestion()).toBeTruthy();
+    });
+    it('Should return false if a string is not a question', () => {
+      expect(actuals.isQuestion3.isQuestion()).toBeFalsy();
+      expect(actuals.isQuestion4.isQuestion()).toBeFalsy();
+    });
+  });
+  describe('words', () => {
+    const expected = {
+      words1: ['Testing', 'of', 'the', 'properties'],
+      words2: ['tsTing', 'of', 'the', 'world', 'is'],
+      words3: ['JINGLe']
+    };
+    const actuals = {
+      words1: 'Testing of the properties ',
+      words2: 'tsTing of the world is',
+      words3: 'JINGLe'
+    };
```

Comment:
**Testing**
You need more test cases to handle unusual punctuations, unusual spacing and special characters.